### PR TITLE
Travis CI automated testing

### PR DESCRIPTION
**Pull request by LasseKS**
Rename to .travis.yml

Test to tests

Changed to tests directory

Added requests

Installing requests

moved install of request

Minor change

Added paramiko

Added simplejson

Inserting copyright statement

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+sudo: false
+dist: trusty
+language: python
+python:
+  - 3.6
+before_install:
+  - pip install requests
+  - pip install paramiko
+  - pip install simplejson
```
Comment (TDI-tenderholt) [inline .travis.yml]: Can you collapse these dependencies into a [`requirements.txt`](https://pip.pypa.io/en/stable/user_guide/#requirements-files) file that can be installed with `pip`?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#
+# This file is part of the Python SDK for PeakInvestigator
+# (http://veritomyx.com) and is distributed under the terms
+# of the BSD 3-Clause license.
+
+sudo: false
+dist: trusty
+language: python
+python:
+  - 3.6
```
Comment (TDI-tenderholt) [inline .travis.yml]: Please add `2.7` for users who haven't made the jump to Python3 and `3.5` since that's the latest in Debian stable.

**Comment by TDI-tenderholt** [comment]
Added a few other minor things that didn't make it into my last round of comments because I did "Start Review" (without "Submit Review") instead of "Add Single Comment".

**Comment by TDI-tenderholt** [comment]
Travis is failing because of a problem related to the abstract base class used in `BaseAction`. See if the `six` module can fix it. See http://six.readthedocs.io/#six.add_metaclass.

**Comment by LasseKS** [comment]
Should now work for Python 2.7 and Python 3

**Comment by TDI-tenderholt** [comment]
Looks good and I'll merge. What's with the "Merge branch 'master' of ... " commit? Did you try a rebase and it was complaining that the two branches diverged?

**Comment by LasseKS** [comment]
No it was not complaining. I just wanted your changes on my branch

**Comment by TDI-tenderholt** [comment]
That makes sense. I think the "cleaner" way of doing it is to fetch changes from the target branch and rebase your branch onto it. Here's what the commit history looks like right now (at least from my machine):

```
*   107917c (HEAD -> master, origin/master) Merge pull request #7 from LasseKS/master
|\  
| *   3360751 Merge branch 'master' of https://github.com/Veritomyx/PeakInvestigator-Python-SDK
| |\  
| |/  
|/|   
* |   ea07f79 Merge pull request #5 from LasseKS/master
|\ \  
| | * 3486771 Use add_metaclas for ABCMeta removed from builtins import property
| | * 863420d Added decorator for metaclass
| | * 4e884ec Added requirements and more python versions
| | * 25c272d Adding travis.yml and travis test
| |/  
| * 0f5489c (Lasse/master) Split test_actions.py into separate actions
|/  
* 06b01f3 Adding a script to run a job with existing files
```

I think the a rebase would look like something from one of our internal projects (I've removed sensitive commit logs):

```
*   9445aba Merge pull request #25 from Veritomyx/PI-320
|\  
| * 5da5459 
| * a2f8210 
| * 112b172 
| * 556b8af 
| * 56adbc0 
| * 4293476
| * 674a632 
| * 0b10e84 
| * d772767 
| * e434c51 
| * b0dfc2a 
| * 45b09a3
|/  
*   4c3ce72 Merge pull request #24 from Veritomyx/PI-304
|\  
| * b22165f 
| * 06cc8c3 
| * 7766562 
```

**Pull request merged**